packages/teams-js/test/public/media.spec.ts

```diff
@@ -702,16 +830,17 @@ describe('media', () => {
       });
 
       it('getMedia call in task frameContext works', async () => {
-        await mobilePlatformMock.initializeWithContext(FrameContexts.task);
-        mobilePlatformMock.setClientSupportedSDKVersion(mediaAPISupportVersion);
-        const mediaOutput: media.Media = new media.Media();
-        mediaOutput.content = '1234567';
-        mediaOutput.mimeType = 'image/jpeg';
-        mediaOutput.format = media.FileFormat.ID;
-        mediaOutput.getMedia(emptyCallback);
-        const message = mobilePlatformMock.findMessageByFunc('getMedia');
-        expect(message).not.toBeNull();
-        expect(message.args.length).toBe(2);
+        mobilePlatformMock.initializeWithContext(FrameContexts.task).then(() => {
```

Comment:
same comment as above

Reply:
Agree this should be fixed. To get unblocked, tabling this issue. Will create a work item.